node/pkg/logscribeconsumer/app.go

```diff
@@ -167,9 +174,15 @@ func extractDbEntry(entry map[string]interface{}) ([]any, error) {
 		return nil, errorsentinel.ErrLogLvlNotExist
 	}
 
+	serviceVal, ok := entry["service"]
+	if !ok {
+		return nil, errorsentinel.ErrLogscribeServiceNotExist
+	}
+
```

Comment:
let's not take service from zerolog msg but from const variable or passed variable when initializing the service. if serviceVal is taken from entry, it means that `service` should be defined in every zerolog messages

Reply:
good point